openshift-kube-apiserver/admission/autoscaling/runonceduration/admission.go

```diff
@@ -16,10 +16,10 @@ import (
 	"k8s.io/utils/integer"
 
 	"github.com/openshift/library-go/pkg/config/helpers"
+	corev1listers "k8s.io/client-go/listers/core/v1"
 	"k8s.io/kubernetes/openshift-kube-apiserver/admission/autoscaling/apis/runonceduration"
 	v1 "k8s.io/kubernetes/openshift-kube-apiserver/admission/autoscaling/apis/runonceduration/v1"
 	"k8s.io/kubernetes/openshift-kube-apiserver/admission/autoscaling/apis/runonceduration/validation"
-	corev1listers "k8s.io/client-go/listers/core/v1"
```

Comment:
this is not generated, is it?

Reply:
same for some other changes. Maybe gofmt output?

Reply:
Added new commit `UPSTREAM: <drop>: Update gofmt` that makes these changes.